db/db_impl.cc

```diff
@@ -664,6 +673,7 @@ size_t DBImpl::EstiamteStatsHistorySize() const {
 
 void DBImpl::PersistStats() {
   TEST_SYNC_POINT("DBImpl::PersistStats:Entry");
+  // 16 digit microseconds timestamp => [Sep 9, 2001 ~ Nov 20, 2286]
```

Comment:
Do we really need microsecond level timestamp in stats?

Reply:
so shall we use millisecond or second for timestamps?
I guess second level is sufficient?